hedera-node/src/main/java/com/hedera/services/ledger/properties/AccountProperty.java

```diff
@@ -80,8 +80,9 @@
 				try {
 					a.setBalance((long) v);
 				} catch (NegativeAccountBalanceException nabe) {
+					// The cast here is to make sonar happy
```

Comment:
It isn't just sonar, Intellij complains about it too. I think the code though should be fixed to handle a potential ClassCastException because we're taking an Object and assuming it will be a long. Perhaps modify the try/catch so that it also catches ClassCastException and deals with a bad cast if it happens.

Reply:
Yeah, will make the changes as suggested.